Description: Allow requesting more than one channel per session.
Author: Andreas Schneider <asn@cryptomilk.org>
Abstract:
 In the 0.5.x series of libssh only one channel request per session
 is possible. This blocks using libssh client sessions which require
 requesting multiple channels on a single SSH connection.
Origin: http://git.libssh.org/projects/libssh.git/commit/src/channels.c?id=e30acdb58a86937e8bece57ce47e272f1106ca55
--- a/src/channels.c
+++ b/src/channels.c
@@ -1951,7 +1951,7 @@
       break;
 
   }
-
+  session->global_req_state = SSH_CHANNEL_REQ_STATE_NONE;
   leave_function();
   return rc;
 error:
